utils/response.js

```diff
@@ -0,0 +1,16 @@
+// Success Response Helper
+export const sendResponse = (res, statusCode, message, data = null) => {
+  return res.status(statusCode).json({
+    success: statusCode >= 200 && statusCode < 300, 
+    message: message,
+    data: data,
+  });
+};
+
+// Error Response Helper
+export const sendErrorResponse = (res, error, statusCode = 500) => {
+  return res.status(statusCode).json({
+    success: false,
+    message: error || "something went wrong",
+  });
+};
```

Comment:
```suggestion
export const sendResponse = (
  res: MedusaResponse,
  success: boolean,
  message: string,
  statusCode: TStatusCode,
  status: TStatus,
  data?: any // server error message or data : {}
): void => {
  res.status(statusCode).json({
    success,
    message,
    status,
    ...(data ? { data } : {}),
  })
}
```

We can make it single generic function to send the success and error response both.